src/utils/guildRole.ts

```diff
@@ -157,6 +157,7 @@ export async function mentionEachUserInMessage({
 }) {
   const batchSize = 5;
   let waitTillNextAPICall = 0;
+  await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Shouldn't this come under try-catch block?

Reply:
It will work the same way. This promise will always get resolved so doesn't matter if you place it inside try-catch block or not.

Reply:
okay got it.